atom/browser/native_window_mac.mm

```diff
@@ -357,6 +357,17 @@ - (void)drawRect:(NSRect)dirtyRect {
     styleMask |= NSTexturedBackgroundWindowMask;
   }
 
+  std::string titleBarStyle = "default";
+  options.Get(switches::kTitleBarStyle, &titleBarStyle);
+
+  if (floor(NSAppKitVersionNumber) >= 1343) {
```

Comment:
For checking OS X version, you can use `base::mac::IsOSYosemiteOrLater()` instead.